Exile.Tanoa/description.ext

```diff
@@ -65,23 +65,41 @@ showHUD[] =
 };
 
 #include "config.cpp"
+#include "ExAdClient\ExAd.cpp"
 
-class CfgRemoteExec
+class CfgFunctions
 {
-    class Functions
-    {
-        mode = 1;
-        jip = 0;
+	#include "ExAdClient\CfgFunctions.cpp"
+};
 
-        class ExileServer_system_network_dispatchIncomingMessage
```

Comment:
Is this not needed anymore with the new cfg remote exec? It'd make sense to add teh new stuff but perhaps not exclude previous settings, like we did for the zombies